.gitignore

```diff
@@ -2,6 +2,6 @@ build
 build_fuzz
 debug
 cex
-aws-c-common
+aws-c-common/*
```

Comment:
why?

Reply:
otherwise reports of all `aws-c-common` files will be ignored by git, and they will not be deployed to gh pages; new expression only ignores everything under `vcc/aws-c-common/`